README.md

```diff
@@ -423,10 +467,38 @@ The example below shows a config including log settings.
 }
 ```
 
+### Cluster Types
+
+The cluster type flag is used to indicate if this is a physical cluster or a Kind cluster.
+
+```yaml
+{
+       "clusterType":"physical",
```

Comment:
There is an option to just make this key something more kind specific to reduce the impact of having to change this configuration for any existing clusters....

Reply:
removed it